Cargo.toml

```diff
@@ -45,6 +45,10 @@ web-time = "1.1.0"
 [workspace.lints.rust]
 rust_2018_idioms = { level = "warn", priority = -1 }
 semicolon_in_expressions_from_macros = "warn"
+unexpected_cfgs = { level = "warn", check-cfg = [
+    'cfg(ruma_identifiers_storage, values("Arc"))', # used by the IdZst macro
+    'cfg(trick_rust_analyzer_into_highlighting_interpolated_bits)', # invalid, generated by xshell::cmd!
```

Comment:
We should think about upgrading xshell or replacing it with another crate, like [cmd_lib](https://crates.io/crates/cmd_lib).

Reply:
Rebased on main to remove this line.